include/aero/actuator/ActuatorGenericSearchFunctor.h

```diff
@@ -124,9 +126,19 @@ struct GenericLoopOverCoarseSearchResults
       // during functor construction i.e. ActuatorBulk, flags, ActuatorMeta,
       // etc.
       //
-      // pointID helps look up data from openfast
       //
-      innerLoopFunctor_(pointId, nodeCoords, sourceTerm, dual_vol, scvIp[nIp]);
+      // for (int actPtInd = 0; actPtInd < innerLoopExtent_; actPtInd ++){
+      //    innerLoopFunctor_(actPtInd, nodeCoords, sourceTerm, dual_vol, scvIp[nIp]);
+      //  }
+      //
+      if (actBulk_.singlePointCoarseSearch_) { 
+        innerLoopFunctor_(pointId, nodeCoords, sourceTerm, dual_vol, scvIp[nIp]);
+      } else {
+        for (int actPtInd = 0; actPtInd < actBulk_.pointCentroid_.extent(0); actPtInd ++){
+          innerLoopFunctor_(actPtInd, nodeCoords, sourceTerm, dual_vol, scvIp[nIp]);
+        }
+      }
```

Comment:
I'm not sure the innerLoopExtent_ idea will work because if singlePointCoarseSearch_=true, the inner functor requires the pointId associated with the index passed as an argument to the generic loop functor. However, if singlePointCoarseSearch_= false, then we need to loop over all actuator points. Is this simple if-else statement appropriate?

Reply:
Yeah I think this is an appropriate use case for if-else.